packages/toolpad-app/src/toolpad/AppEditor/PageLayoutEditor.tsx

```diff
@@ -0,0 +1,53 @@
+import { MenuItem, TextField } from '@mui/material';
+import * as React from 'react';
+import { AppDom, PageLayoutMode, PageNode, setNodeNamespacedProp } from '../../appDom';
+import { useDomApi } from '../AppState';
+
+interface PageLayoutEditorProps {
+  node: PageNode;
+}
+
+const PAGE_LAYOUT_OPTIONS: { value: PageLayoutMode; label: string }[] = [
+  { value: 'container', label: 'Container' },
+  { value: 'fluid', label: 'Fluid' },
+];
+
+export default function PageLayoutEditor({ node }: PageLayoutEditorProps) {
```

Comment:
This suggestion is completely optional, so don't worry about it: but this control might work better as a toggle as there's only 2 options? Maybe https://mui.com/material-ui/react-toggle-button/
Anyway no need to do this in this PR unless you really want to - we plan to make the same adjustment for the page `display` attribute too.